spec/lib/open_api/serializing_spec.rb

```diff
@@ -25,5 +25,14 @@ def to_hash
 
       expect(model.new.as_json).to eq("a" => 1, "b" => 2, "secret" => "value")
     end
+
+    it "works" do
```

Comment:
Ehhh can we put something a bit better here?

Ideally this would read `#api_version_from_prefix parses the correct version`